app.js

```diff
@@ -14,12 +14,30 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then(beersFromApi => {
+    res.render('beers', {beersFromApi});
+  })
+  .catch(error => console.log(error));
+});
+
+app.get('/randombeer', (req, res) => {
+  punkAPI
+  .getRandom()
+  .then(responseFromAPI => {
+    res.render('randombeer', {responseFromAPI});
```

Comment:
This is correct. Another way to do it is to just pass `responseFromAPI[0]` to res.render since the `.getRandom()` will always return an array of one element and so we can skip having to use `#each` in the hbs.